frontend/package.json

```diff
@@ -43,7 +43,8 @@
     "vue-tsc": "^2.0.11"
   },
   "dependencies": {
-    "@maplibre/maplibre-gl-directions": "^0.7.0",
+    "@axe-core/playwright": "^4.9.0",
+    "@maplibre/maplibre-gl-directions": "^0.6.1",
```

Comment:
Note that the MapLibre downrade's happening here, which should be avoided 🤔

Reply:
I can handle this :)

Reply:
I think this was from fixing a merge conflict mistake. My bad.